app/dashboard/src/views/dashboard/member/edit/EditMemberGeneralView.vue

```diff
@@ -173,6 +173,9 @@ export default class EditMemberGeneralView extends Mixins(NavigationMixin) {
                 memberDetails.records.push(Record.create({
                     type: RecordType.NoPictures
                 }))
+                memberDetails.records.push(Record.create({
+                    type: RecordType.OnlyGroupPictures
+                }))
```

Comment:
Voor nieuwe leden mogen we niet automatisch 'toestemming voor groepsfoto's' aanzetten. Dan is er geen impliciete toestemming gegeven omdat het al aangevinkt was.